xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentSigns_version.xsd

```diff
@@ -328,6 +343,32 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="other"/>
 		</xsd:restriction>
 	</xsd:simpleType>
+
+	<xsd:simpleType name="AudioAnnoucementTypeEnumeration">
```

Comment:
@nick-knowles @Aurige  I am seeing completely different values in master, while @nick-knowles 's famous modification changes suggest they should be there.

```xml
        <xsd:simpleType name="AudioAnnouncementTypeEnumeration">
                <xsd:annotation>
                        <xsd:documentation>Allowed values for AudioAnnouncements </xsd:documentation>
                </xsd:annotation>
                <xsd:restriction base="xsd:NMTOKEN">
                        <xsd:enumeration value="onDemand"/>
                        <xsd:enumeration value="automatic"/>
                </xsd:restriction>
        </xsd:simpleType>
```

Reply:
yes, the one from the PR is the one agreed (should be in the doc) and is more comprehensive